server/routerlicious/packages/services-ordering-rdkafka/src/rdkafkaBase.ts

```diff
@@ -90,4 +91,12 @@ export abstract class RdkafkaBase extends EventEmitter {
 			});
 		});
 	}
+
+    protected error(error: any, restartOnError: boolean = false) {
```

Comment:
nit: I think the tabbing/spacing is off. It should be 2 spaces

Reply:
Thanks Gary! I think it's fixed now